docs/IPFS.md

```diff
@@ -0,0 +1,67 @@
+# IPFS
+For an overview about IPFS, visit the [IPFS project site](https://ipfs.tech/).
```

Comment:
You will need to extend `.github/scripts/spellcheck.words` with IPFS and IPNS at least, to please the spellchecker.

Reply:
Thanx for that pointer, will update that one too.